pandas/io/formats/style_render.py

```diff
@@ -145,6 +146,23 @@ def render(self, **kwargs) -> str:
         d.update(kwargs)
         return self.template_html.render(**d)
 
+    def _render_latex(self, **kwargs) -> str:
```

Comment:
lets rename render -> render_html? (you may need to keep render as an alias for backwards compat but that's ok)

Reply:
see #41123